src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/LanguageVersionExtensions.cs

```diff
@@ -7,10 +7,6 @@ namespace Microsoft.CodeAnalysis.CSharp.Shared.Extensions
     internal static class LanguageVersionExtensions
     {
         public static bool IsCSharp9OrAbove(this LanguageVersion languageVersion)
-#if CODE_STYLE
-            => languageVersion >= (LanguageVersion)900;
-#else
             => languageVersion >= LanguageVersion.CSharp9;
-#endif
```

Comment:
note: i woudl expect to just remove this method at thsi point and inline tha tcheck.

Reply:
I'm neutral on this. Could be done as a follow-up.